sycl/test-e2e/Matrix/joint_matrix_bfloat16_packedB.cpp

```diff
@@ -13,12 +13,5 @@
 
 #include "common.hpp"
 
-using namespace sycl;
-using namespace sycl::ext::oneapi::experimental::matrix;
-
 #define SG_SZ 16
```

Comment:
you should use `get_sg_size`  to get rid of this macro here

Reply:
This is not the point of this PR